src/app/slice.js

```diff
@@ -17,7 +18,10 @@ const initialState = {
   pages: [],
   randomArchives: [],
   renderedPages: [],
-  searchArchives: [],
+  searchArchives: {
```

Comment:
Turning this into an object breaks one of the slice updating functions. Let's keep this as an array and if you really need to save the total to the state then just add it to the slice as its own value like `totalArchives` or something along those lines.